src/components/node-list/node-list-items.js

```diff
@@ -148,15 +152,15 @@ export const getFilteredModularPipelineItems = createSelector(
       (modularPipeline) => ({
         ...modularPipeline,
         type: 'modularPipeline',
-        visibleIcon: IndicatorIcon,
-        invisibleIcon: IndicatorOffIcon,
+        visibleIcon: VisibleIcon,
+        invisibleIcon: InvisibleIcon,
         active: false,
         selected: false,
         faded: false,
         visible: true,
         disabled: false,
-        unset: !modularPipeline.enabled,
-        checked: modularPipeline.enabled,
+        unset: false,
```

Comment:
thanks for raising this - this change is mainly due to the change of the modular pipeline item now being a 'element' type item rather than 'filter' type similar to tag, where the unset field is basically only used in a scenario similar to setting an individual tag, which does not exist anymore with this iteration. 

( i.e `modularPipeline.enabled` for each modular pipeline is not controlled by setting in the same fashion as setting a filter on or off) hence I am setting this false by default to avoid confusion, in a similar fashion to how the node items are constructed ( refer to line 225 to see how we have set the `unset` field for node items as default.)